themes/prism-a11y-dark.css

```diff
@@ -9,6 +9,7 @@ pre[class*="language-"] {
 	color: #f8f8f2;
 	background: none;
 	font-family: Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
+	font-size: 1em;
```

Comment:
This is the only one that has the font-size set to 1em vs what you did elsewhere:

```css
pre > code[class*="language-"] {
	font-size: 1em;
}
```

Do you remember why?

Reply:
No idea.

This is only one which did not declare a `font-size` for which I added this.

Probably a mistake? I'll remove it for now.